### ISSUE TITLE
Querying items fails when passing NaN

### ISSUE THREAD

**Issue by weyert** [balderdashy/sails#2204]
If you want to query items via the API it will fail when you try to pass NaN as returned by the actionUtil.parseLimit()-method.

**Comment by CWyrtzen** [comment]
@weyert did you find a resolution or work-around? is this still happening with the latest release?

**Comment by sailsbot** [comment]
Thanks for posting, @weyert.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
